stream.go

```diff
@@ -661,7 +668,13 @@ func (cs *clientStream) withRetry(op func(a *csAttempt) error, onSuccess func())
 	for {
 		if cs.committed {
 			cs.mu.Unlock()
-			return op(cs.attempt)
+			err := op(cs.attempt)
+			if err != nil {
+				if nse, ok := err.(*transport.NewStreamError); ok {
```

Comment:
Can this just `return toRPCErr(err)`?

Reply:
Yes, it can now.  I think I moved that unwrapping there after I wrote this.  Fixed.